Userland/Services/WebContent/ConnectionFromClient.h

```diff
@@ -90,6 +94,9 @@ class ConnectionFromClient final
     WeakPtr<JS::Interpreter> m_interpreter;
     OwnPtr<WebContentConsoleClient> m_console_client;
     JS::Handle<JS::GlobalObject> m_console_global_object;
+
+    HashMap<int, NonnullRefPtr<Web::FileRequest>> m_requested_files {};
+    int last_id {};
```

Comment:
Please default-initialize primitive variables
```suggestion
    int last_id { 0 };
```

Reply:
They are default-initialized, but I can explicitly write it